group_vars/all/dcos.yaml.example

```diff
@@ -2,6 +2,8 @@
 dcos:
   download: "https://downloads.dcos.io/dcos/stable/1.13.4/dcos_generate_config.sh"
   download_checksum: "sha256:55b096236a6de1a40259e6d8a8a21861c5333cbe2b9ba3e274d5de1534b095dc"
+  download_win: "https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh"
```

Comment:
i'm a bit concerned stating testing here. For master it should be stable. 

Also question is if we could have this in a single deployment. I know this was already an internal discussion.

Reply:
Hi @fatz, 
The concern is understandable, however for now "stable" version is not ready. ETA is end of November2019, it will be for sure. Therefore I will update in the example config the stable version.
 
About single vs separated Windows installer - as per initial design there was requirement to keep Windows installer separately from Linux installer, because of 
1) Linux installer takes 1GB+ of size and it's too much to increase the size
2) Most of Linux consumers doesn't need Windows installer
3) Windows installer is produced based on separated CI build, which involves Windows agents and doesn't involve Linux systems.

Reply:
> The concern is understandable, however for now "stable" version is not ready. ETA is end of November2019, it will be for sure. Therefore I will update in the example config the stable version.
Yeah for non stable we still have the windows branch. That should be used until we finally release it and offer a stable version.

> Linux installer takes 1GB+ of size and it's too much to increase the size
well the stated file is just 109MB thats fair enough adding to DC/OS

> Most of Linux consumers doesn't need Windows installer
That is true but given the size of the current genconf it doesn't really matter. The risk of conflicts and a user messing up versions here is pretty high

> Windows installer is produced based on separated CI build, which involves Windows agents and doesn't involve Linux systems.
well its still DC/OS and this adds a bunch of complexity to the current structure which is something we should handle und take care of not the user.
That being said the usual genconf already contains the serve folder so we just need to add the windows packages into that folder and it would solve this right away